diff -uNr origin/lib/jubatus/anomaly/client.rb for_jubatus/lib/jubatus/anomaly/client.rb
--- origin/lib/jubatus/anomaly/client.rb	2013-03-13 17:36:43.693042538 +0900
+++ for_jubatus/lib/jubatus/anomaly/client.rb	2013-03-13 17:37:30.217042541 +0900
@@ -13,6 +13,9 @@
   def initialize(host, port)
     @cli = MessagePack::RPC::Client.new(host, port)
   end
+  def get_client
+    @cli
+  end
   def get_config(name)
     @cli.call(:get_config, name)
   end
diff -uNr origin/lib/jubatus/classifier/client.rb for_jubatus/lib/jubatus/classifier/client.rb
--- origin/lib/jubatus/classifier/client.rb	2013-03-13 17:36:43.705042540 +0900
+++ for_jubatus/lib/jubatus/classifier/client.rb	2013-03-13 17:37:30.229042539 +0900
@@ -13,6 +13,9 @@
   def initialize(host, port)
     @cli = MessagePack::RPC::Client.new(host, port)
   end
+  def get_client
+    @cli
+  end
   def get_config(name)
     @cli.call(:get_config, name)
   end
diff -uNr origin/lib/jubatus/graph/client.rb for_jubatus/lib/jubatus/graph/client.rb
--- origin/lib/jubatus/graph/client.rb	2013-03-13 17:36:43.717042539 +0900
+++ for_jubatus/lib/jubatus/graph/client.rb	2013-03-13 17:37:30.241042539 +0900
@@ -13,6 +13,9 @@
   def initialize(host, port)
     @cli = MessagePack::RPC::Client.new(host, port)
   end
+  def get_client
+    @cli
+  end
   def get_config(name)
     @cli.call(:get_config, name)
   end
diff -uNr origin/lib/jubatus/recommender/client.rb for_jubatus/lib/jubatus/recommender/client.rb
--- origin/lib/jubatus/recommender/client.rb	2013-03-13 17:36:43.725042540 +0900
+++ for_jubatus/lib/jubatus/recommender/client.rb	2013-03-13 17:37:30.249042540 +0900
@@ -13,6 +13,9 @@
   def initialize(host, port)
     @cli = MessagePack::RPC::Client.new(host, port)
   end
+  def get_client
+    @cli
+  end
   def get_config(name)
     @cli.call(:get_config, name)
   end
diff -uNr origin/lib/jubatus/regression/client.rb for_jubatus/lib/jubatus/regression/client.rb
--- origin/lib/jubatus/regression/client.rb	2013-03-13 17:36:43.733042539 +0900
+++ for_jubatus/lib/jubatus/regression/client.rb	2013-03-13 17:37:30.257042539 +0900
@@ -13,6 +13,9 @@
   def initialize(host, port)
     @cli = MessagePack::RPC::Client.new(host, port)
   end
+  def get_client
+    @cli
+  end
   def get_config(name)
     @cli.call(:get_config, name)
   end
diff -uNr origin/lib/jubatus/stat/client.rb for_jubatus/lib/jubatus/stat/client.rb
--- origin/lib/jubatus/stat/client.rb	2013-03-13 17:36:43.745042539 +0900
+++ for_jubatus/lib/jubatus/stat/client.rb	2013-03-13 17:37:30.265042540 +0900
@@ -13,6 +13,9 @@
   def initialize(host, port)
     @cli = MessagePack::RPC::Client.new(host, port)
   end
+  def get_client
+    @cli
+  end
   def get_config(name)
     @cli.call(:get_config, name)
   end
